packages/client/hmi-client/src/services/calibrate-workflow.ts

```diff
@@ -56,22 +55,8 @@ export const setupCsvAsset = async (dataset: Dataset): Promise<CsvAsset | undefi
 	// FIXME: We are setting the limit to -1 (i.e. no limit) on the number of rows returned.
```

Comment:
nitpick you can feel completely free to ignore
Can we change name from `setupCsvAsset` to something more accurate like `getCsvFromDataset`

Reply:
I'll do that in a different PR since it will change multiple import statements as well!